packages/mwp-app-route-plugin/src/route.test.js

```diff
@@ -30,6 +30,22 @@ describe('onPreResponse.method', () => {
 		expect(errorMarkup).toContain(errorMessage);
 		expect(replyObj.code).toHaveBeenCalledWith(errorCode);
 	});
+	it('returns X-Meetup-External-Track and X-Meetup-External-Track-Url headers if _xtd query param exists', () => {
```

Comment:
do you want a test to make sure these headers are _not_ set when the query param is _not_ present? If someone accidentally broke the `if (trackingParam) {` check, this test would pass even though it wouldn't be working as intended.